controllers/foos.controller.js

```diff
@@ -38,7 +38,7 @@ const users = [
   { fName: "Gellert", lName: "Grindelwald" },
 ];
 
-const requestTypes = ["receive", "provide", "deliver", "receive"];
+const requestTypes = ["receive", "provide"];
```

Comment:
Could we add these back? I'm reasonably confident we'll need this if we want to support shipping/shippers/items.

Reply:
Things may have changed without me knowing so let me know if they have.

Reply:
sure I was just testing the distance feature using the foos controller. The overall models have not been changed